app/src/main/java/com/hydrofish/app/ui/composables/tabs/HomeScreen.kt

```diff
@@ -140,3 +173,319 @@ fun ReusableDrinkButton(waterAmt: Float, hydroFishViewModel: HydroFishViewModel
         }
     }
 }
+
+
+object FishPaths {
+    val fish_v1 = PathParser().parsePathString(
+        """
+        M2,45C2,38.95 15.35,20.28 23,15.78C26.94,13.46 28.99,9.66 33.56,8C37.42,6.6 42.97,4.22 47,4C54.06,3.61 60.61,2 68,2C71.5,2 75,2 78.5,2C82.37,2 85.43,3.8 89,4C97.77,4.49 106.76,13.41 113.78,18C120.75,22.56 126.74,29.19 134,33.22C138.38,35.65 144.84,44.49 147.22,49C148.45,51.31 150.06,54.58 151.56,56.5C153.17,58.57 156,65.62 156,63
+      
+            """.trimIndent()
+    ).toNodes()
+    val fish_v2 = PathParser().parsePathString(
+        "M2,51C5.97,51.44 8.09,56.57 12,57C16.32,57.48 20.51,60.75 25,61C28.72,61.21 32.39,63 36.22,63C40.89,63 45.56,63 50.22,63C57.12,63 67.71,61.02 74,58.22C77.5,56.67 81.36,56.6 85,54.78C88.21,53.17 91.78,51.93 95,50.22C101.75,46.65 108.41,42.44 115,38.78C121.29,35.29 128.63,31.93 133.78,26.78C138.03,22.53 142.37,18.51 146.22,14.22C147.95,12.3 150.05,10.7 151.78,8.78C153.72,6.61 156.76,4.49 158,2"
+    ).toNodes()
+    val fish_v3 = PathParser().parsePathString(
+        "M2,2C2,12.866 5,23.851 5,35C5,45.286 8.602,57.797 4,67"
+    ).toNodes()
+    val fish_v4 = PathParser().parsePathString(
+        "M2,2C2,10.323 5,18.017 5,26.5C5,33.508 8,40.756 8,48"
+    ).toNodes()
+    val fish_v5 = PathParser().parsePathString(
+        "M2,2C2,3.7633 3.5079,3.754 2,3"
+    ).toNodes()
+}
+
+
+object SharkPaths {
```

Comment:
Please move the image motion paths to a separate file to keep HomeScreen.kt clean and readable.

Reply:
Finished moving paths.